### PR TITLE
Improve Deploying glue jobs documentation following user testing

### PR DESCRIPTION
**Pull request by b-dalton**

### CODE REVIEW

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md [hunk 1]
```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
...
-you can find the automatically generated script by navigating to the `Script` tab when viewing your job in Glue Studio.
+1. Navigate to the [jobs directory][jobs-directory] in the Data Platform Project and open your department folder.
+1. Click `Add file` and then `Create new file`.
+1. Name your file.
+   This should be the same name you gave your job in Glue Studio, but should be with all **lowercase letters**, **words separated by underscores**, and with the extension **`.py`** at the end.
+   e.g. `address_cleaning_housing_repairs.py`.
+   - _Remember this name as you will need it for the following section._
+1. Open your Glue job in Glue Studio and copy the auto generated **Python** script by navigating to the `Script` tab.
+
+_Note: This is only applicable if your Glue job was created using the Glue Studio UI._
```
Comment (elcorbs) [inline docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md:27]: Maybe just change this to "If you created your job in Glue Studio, ... " instead of having the note underneath

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md [hunk 2]
```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
...
-- Click `Add file` and then `Create new file`
-- Name your file, using underscores as separators e.g. `address_cleaning_housing_repairs`, and then paste your code in field below
-- Follow [the instructions here][committing-changes] to commit your new file to the project
-
 ### 2. Copy an existing Glue job module block or use the example below and populate the necessary variables
 
-Once the Pull Request to add your script has been approved, and you have confirmed that your script is in the Data Platform Project,
-you can proceed with this step.
-In this step, you will be creating a module which will deploy your Glue job (and crawler) to the Data Platform Project.
+Once the Pull Request to add or update your script has been approved, and you have confirmed that your script is in the Data Platform Project,
```
Comment (elcorbs) [inline docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md:39]: I guess they don't need to wait for this in order to continue to making a second pull request?

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md [hunk 3]
```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
...
-    23-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf
-    ```
-    _Refer to the [example](#example-module-block) below to get started._
-- Click `edit` or the pencil icon and copy and paste an existing module block.
-- When updating the module:
-  - Do not change any spacing before the `=`
-  - Do not remove the `"quotation marks"` around any values which use them
-  - When adding a variable that does not already exist, paste the variable name and then intended value after the `=`
-    and ensure that all equal signs (`=`) line up as shown in the example
+In this section, you will be creating a module (or updating an existing one if you need to make changes to an existing Glue job) which will deploy your Glue job (and crawler) to the Data Platform Project.
```
Comment (elcorbs) [inline docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md:42]: A module won't mean anything to most people. Maybe reword to something like "you will be writing the code which will deploy..."

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md [hunk 4]
```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
...
+1. Open `25-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf`. e.g. `25-aws-glue-job-parking.tf`
+
+   - _If this file does not exist, create a new terraform file for your department by clicking `Add file` then `Create new file`.
+     The name of the new file should be:_
+
+   ```
+   25-aws-glue-jobs-<YOUR-DEPARTMENT-NAME>.tf
+   ```
+
+   - _Refer to the [example](#example-module-block) below to get started._
```
Comment (elcorbs) [inline docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md:57]: I would add this example text to the preamble bit of this section instead of having it in this step.

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md [hunk 5]
```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
...
+
+1. Click `edit` or the **pencil icon** (:pencil2:) and copy and paste an existing module block.
+   An example of what a module block looks like can be seen [here][project-module-example].
 
-Update the `module` name using the name convention `<job_name>_<department_name>`, for example "liberator_pcn_denormalisation_housing_repairs".
-The module name must be all lowercase with words separated by underscores. Ensure it's unique to all other module names in this file.
+1. Update the `module` name using the name convention `<job_name>_<department_name>`, for example: `"liberator_pcn_denormalisation_housing_repairs"`.
+
+   - The `job_name` must be the same as the script name you specified in the previous section (without the `.py` extension).
```
Comment (elcorbs) [inline docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md:64]: This step is talking about the module name, maybe move this line to be with the section where you specify the job name.

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md [hunk 6]
```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
...
+1. #### Update or add your input variables.
+   For this section, you will need the following values from your Glue job.
+   You can find them by navigating to the `Job Details` tab in Glue Studio.
+   - **Name**
+   - **Description**
+   - **Job parameters**. Expand the `Advanced properties` section and scroll down to `Job parameters`.
 
 #### The following input variables are required:
 
+_Note: If you've copied an existing module block from your department folder then you won’t need to change the **source** and **department** variables and will only need to update the **job_name** variable._
```
Comment (elcorbs) [inline docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md:77]: Also won't need to change the `helper_module_key` &  `pydeequ_zip_key`

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md [hunk 7]
```diff
@@ -16,95 +16,129 @@ If you are still testing or prototyping your script, it is recommended that you
...
-  One of the following variables must be populated.
+
+  One of the following variables must be populated: **script_name** OR **script_s3_object_key**.
+
   If you are adding a new script to only be used for one glue job you should provide a value for **script_name** and leave the second blank.
   If your script file is already saved in S3 you should provide **script_s3_object_key**.
-  - **script_name** : Name of the Glue job script. Set this to the name of the script file you created in [step 1][adding-script]  
-    e.g. `"address_cleaning_housing_repairs"`. This file must be saved within your departmental folder.
+
+  - **script_name** : Name of the Glue job script. Set this to the name of the script file you created in [step 1][adding-script].
```
Comment (elcorbs) [inline docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md:100]: Maybe mention it's the name of the script file, without `.py`

File: docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md (hunk 5 above)
Comment (b-dalton) [reply docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md]: I've made it clearer that it refers to the `<job_name>` in module name: `<job_name>_<department_name>`

Comment (b-dalton) [reply docs/playbook/transforming-data/using-aws-glue/002-deploy-glue-jobs.md]: And also added that line to the `job_name` variable
